AirCasting/Graph/Graph.swift

```diff
@@ -90,13 +91,37 @@ class UI_PollutionGraph: UIView {
         dataSet.setColor(UIColor(.white))
         dataSet.mode = .linear
         dataSet.lineWidth = 4
+        
+        if !didMoveOrScaleGraph && isSessionMobile {
+            zoomoutToThirtyMinutes(dataSet: dataSet)
+        }
+    }
+    
+    func zoomoutToThirtyMinutes(dataSet: LineChartDataSet) {
```

Comment:
sepperate the private methods from public to avoid confusion of API readability.